nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/LoggableKey.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.runtime.test.runner;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.lang.annotation.ElementType;
+
+/**
+ * Represent the key for a given loggable, a {@link Loggable} can be defined on {@code Class} or {@Method}.
+ *
+ * @since 11.1
+ */
+public class LoggableKey {
+    protected final ElementType type;
+
+    protected final String loggerName;
+
+    public LoggableKey(ElementType type, String loggerName) {
+        this.type = type;
+        this.loggerName = loggerName;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o)
+            return true;
+
+        if (o == null || getClass() != o.getClass())
+            return false;
+
+        LoggableKey that = (LoggableKey) o;
+
+        return new EqualsBuilder().append(type, that.type).append(loggerName, that.loggerName).isEquals();
```

Comment:
Could you use `EqualsBuilder.reflectionEquals(this, o)` instead? Same thing for `hashCode`, you can find an example in `BulkStatus`.